<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositsFrame</class>
 <widget class="QFrame" name="DepositsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="m_overviewWalletFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Deposits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Locked:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="m_lockedDepositLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="m_tickerLabel1">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Unlocked:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="m_unlockedDepositLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="m_tickerLabel2">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="m_totalDepositLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="m_tickerLabel3">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="m_withdrawButton">
        <property name="text">
         <string>Withdraw</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_addDepositFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="m_amountSpin">
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="m_nativeTimeLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_timeSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="m_interestLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>534</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="m_depositButton">
        <property name="text">
         <string>Deposit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_feeTextLabel">
        <property name="text">
         <string>Deposit fee:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="m_feeLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="m_depositView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Balance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_balanceLabel">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_depositButton</sender>
   <signal>clicked()</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_withdrawButton</sender>
   <signal>clicked()</signal>
   <receiver>DepositsFrame</receiver>
   <slot>withdrawClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_depositView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>showDepositDetails(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>timeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_amountSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>depositClicked()</slot>
  <slot>withdrawClicked()</slot>
  <slot>showDepositDetails(QModelIndex)</slot>
  <slot>timeChanged(int)</slot>
  <slot>depositParamsChanged()</slot>
 </slots>
</ui>
